pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1890,6 +1890,8 @@ type StorageScopeSpec struct {
 	Nodes []Node `json:"nodes,omitempty"`
 	// +optional
 	UseAllNodes bool `json:"useAllNodes,omitempty"`
+	// +optional
+	SkipApplyAllPlacement bool `json:"skipApplyAllPlacement,omitempty"`
```

Comment:
What about this name? Then in the comments and documentation we could describe in more detail and give an example of the behavior. It is a complex feature in any case, so the naming will be difficult on this setting.
```suggestion
	OnlyApplySpecificOSDPlacement bool `json:"onlyApplySpecificOSDPlacement,omitempty"`
```

Reply:
done.